pom.xml

```diff
@@ -4,10 +4,9 @@
   <artifactId>spark-job-server-client</artifactId>
   <version>1.0.0</version>
   <name>spark-job-server-client</name>
-  <description>Java Lib for Spark Job Server Rest API</description>
-  
-  <dependencies>
+	<description>Java Lib for Spark Job Server Rest API</description>
 
```

Comment:
The content seems doesn't change, so this file can be checkout with the old format